src/OpenTelemetry.Api/Internal/SemanticConventions.cs

```diff
@@ -110,5 +110,15 @@ internal static class SemanticConventions
         public const string AttributeExceptionType = "exception.type";
         public const string AttributeExceptionMessage = "exception.message";
         public const string AttributeExceptionStacktrace = "exception.stacktrace";
+
+        // NEW v1.21.0 Http Semantic Conventions
```

Comment:
What does this mean?

Reply:
I was asked to separate the new attributes from the others and give it a descriptive comment.
The word "new" isn't needed here, sorry for the confusion.
I've updated this comment in the AspNetCore PR but forgot to update it here.

Reply:
> I was asked to separate the new attributes from the others and give it a descriptive comment.

What does v1.21.0 mean?

> The word "new" isn't needed here

👍